libbeat/reader/readfile/metafields.go

```diff
@@ -41,10 +41,9 @@ func NewFilemeta(r reader.Reader, path string, offset int64) reader.Reader {
 func (r *FileMetaReader) Next() (reader.Message, error) {
 	message, err := r.reader.Next()
 
-	r.offset += int64(message.Bytes)
```

Comment:
Can you add a comment in here noting that `log.offset` should be the start of the reported line, and not the end? Let's document what the behaviour needs to be and why in the code itself.

Reply:
Looks like I was too late :) The commit message will explain it at least.